contracts/integration-tests/test/deployment_manager_test.ts

```diff
@@ -0,0 +1,176 @@
+            const feeMultiplier = await deploymentManager.exchange.protocolFeeMultiplier.callAsync();
+            expect(feeMultiplier).bignumber.to.be.eq(DeploymentManager.protocolFeeMultiplier);
+        });
+    });
+
+    describe('staking', () => {
+        it('should be owned by the asset proxy owner', async () => {
+            const stakingOwner = await deploymentManager.staking.stakingProxy.owner.callAsync();
+            expect(stakingOwner).to.be.eq(deploymentManager.assetProxyOwner.address);
+        });
+
+        it('should have authorized the asset proxy owner in the staking proxy', async () => {
+            const isAuthorized = await deploymentManager.staking.stakingProxy.authorized.callAsync(
+                deploymentManager.assetProxyOwner.address,
+            );
+            expect(isAuthorized).to.be.true();
+        });
+
+        it('should have registered the exchange in the staking proxy', async () => {
+            const isValid = await deploymentManager.staking.stakingProxy.validExchanges.callAsync(
+                deploymentManager.exchange.address,
+            );
+            expect(isValid).to.be.true();
+        });
+
+        it('should have registered the read-only proxy in the staking proxy', async () => {
+            const readOnlyProxy = await deploymentManager.staking.stakingProxy.readOnlyProxy.callAsync();
+            expect(readOnlyProxy).to.be.eq(deploymentManager.staking.readOnlyProxy.address);
+        });
+
+        it('should have registered the staking contract in the staking proxy', async () => {
+            const stakingContract = await deploymentManager.staking.stakingProxy.stakingContract.callAsync();
+            expect(stakingContract).to.be.eq(deploymentManager.staking.stakingLogic.address);
+        });
+
+        it('should have registered the weth contract in the staking contract', async () => {
+            const weth = await deploymentManager.staking.stakingWrapper.wethAddress.callAsync();
+            expect(weth).to.be.eq(deploymentManager.tokens.weth.address);
+        });
+
+        it('should have registered the zrx vault in the staking contract', async () => {
+            const zrxVault = await deploymentManager.staking.stakingWrapper.zrxVault.callAsync();
+            expect(zrxVault).to.be.eq(deploymentManager.staking.zrxVault.address);
+        });
+
+        it('should have registered the staking proxy in the zrx vault', async () => {
+            const stakingProxy = await deploymentManager.staking.zrxVault.stakingProxyAddress.callAsync();
+            expect(stakingProxy).to.be.eq(deploymentManager.staking.stakingProxy.address);
+        });
+
+        it('should have correctly set the params', async () => {
+            const params = await deploymentManager.staking.stakingWrapper.getParams.callAsync();
+            expect(params).to.be.deep.eq(
+                [10 * 60 * 60 * 24, (90 * constants.PPM_DENOMINATOR) / 100, 100 * 1e18, 10, 1, 2].map(
```

Comment:
Is there a way we can just import these from `@0x/contracts-staking/test/utils/constants.ts` so they're less likely to get out of sync?